package.json

```diff
@@ -19,12 +19,10 @@
     "prettier": "prettier --write \"**/*.{scss}\"",
     "prepare": "yarn build"
   },
-  "dependencies": {
-    "archiver": "^3.0.0",
-    "git-branch": "^2.0.1"
-  },
   "devDependencies": {
+    "archiver": "^3.0.0",
     "fs-extra": "^7.0.0",
+    "git-branch": "^2.0.1"
```

Comment:
Syntax error here - gotta have a comma at the end of this line.

Reply:
Thanks @akx! That's what I get for editing through GitHub UI 😂